internal/trigger/http/rest.go

```diff
@@ -56,15 +57,17 @@ func (trigger *Trigger) requestHandler(writer http.ResponseWriter, r *http.Reque
 	contentType := r.Header.Get(clients.ContentType)
 
 	if contentType != clients.ContentTypeJSON && contentType != clients.ContentTypeCBOR {
-		trigger.logging.Debug("HTTP content type not supported", clients.ContentType, contentType)
+		trigger.logging.Error("HTTP content type not supported", clients.ContentType, contentType)
```

Comment:
I'd rather keep this Debug. I feel like we've been here before :) Want to prevent a bot from spamming the logs and sending an invalid content type header because most likely will be set to Error in production.

Reply:
Actually this check is redundant with the checks in ProcessMessage. I will remove this one. But that just moves it...

Logging REST errors as debug is not consistent with the rest of edgex. Example:
https://github.com/edgexfoundry/edgex-go/blob/master/internal/core/data/router.go#L141